<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QWidget" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>630</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>630</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Домашнее окно администратора</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/icons/book_text.svg</normaloff>:/icons/book_text.svg</iconset>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>630</width>
     <height>480</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>630</width>
     <height>480</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>630</width>
     <height>480</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Добавить агента</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>361</width>
       <height>321</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="surnameLineEdit">
        <property name="placeholderText">
         <string>Фамилия</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameLineEdit">
        <property name="placeholderText">
         <string>Имя</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="patronymicLineEdit">
        <property name="placeholderText">
         <string>Отчество</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="phoneLineEdit">
        <property name="placeholderText">
         <string>Номер телефона</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addAgentButton">
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Удалить агента</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QTableWidget" name="agentsViewTable">
       <column>
        <property name="text">
         <string>id</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Фамилия</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Имя</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Отчество</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Телефон</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="refreshButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/icons/refresh_alt.svg</normaloff>:/icons/refresh_alt.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/icons/cross_circle.svg</normaloff>:/icons/cross_circle.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Выйти</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
